urdf/package.xml

```diff
@@ -24,12 +24,12 @@
   <build_depend>console_bridge</build_depend>
   <build_depend>urdfdom</build_depend>
   <build_depend>urdfdom_headers</build_depend>
-  <build_depend>TinyXML</build_depend>
+  <build_depend>tinyxml</build_depend>
   <build_depend>tinyxml_vendor</build_depend>
 
   <exec_depend>urdfdom</exec_depend>
-  <exec_depend>urdfdom-headers</exec_depend>
-  <exec_depend>TinyXML</exec_depend>
+  <exec_depend>urdfdom_headers</exec_depend>
```

Comment:
I'm not sure about this one. What's the situation, `urdfdom-header` points to an upstream deb via rosdep which we cannot use in ROS 2? So this change allows us to use a fork of `urdfdom_headers`?

If that is the case. I guess that's the right thing to do, but it's an unfortunately subtle change, which I hope we remember to change back at some point.

I guess my unease could be put to rest if we have a comment on it, saying it needs to be resolved at some point.

Here's some related stuff:

- https://github.com/ros2/urdfdom_headers/pull/1#pullrequestreview-36626436
- https://github.com/ros/urdfdom_headers/pull/34